Spanish translation

emmanuellfc opened the pull request.

emmanuellfc committed:
- Start of Spanish translation.

emmanuellfc committed:
- Start of Spanish translation.

emmanuellfc committed:
Merge branch 'i18n' of https://github.com/emmanuellfc/morpho-manual-translation into i18n

emmanuellfc committed:
- Work in progress

emmanuellfc committed:
- Work in progress with Spanish translation: 90% complete

To Do: Once completed the translation review the whole manual and find error and fix it.

emmanuellfc committed:
- Work in progress with Spanish translation: 100% complete

To Do: Once completed the translation review the whole manual and find errors and fix them.

joshichaitanya3 commented:
This is an incredible contribution, thank you so much @emmanuellfc !! I am now going through it page by page to catch any formatting errors etc. Will get back soon!

emmanuellfc committed:
- Work in progress with Spanish translation: 100% complete

Done: Once completed the translation review the whole manual and find errors and fix them.

joshichaitanya3 commented:
Thanks for the fixes, @emmanuellfc! I’ll merge this now and create new Issues if something else comes up. 

Can’t wait for this to go live!
